packages/flutter_test/lib/src/finders.dart

```diff
@@ -284,6 +285,13 @@ abstract class Finder {
   /// matched by this finder.
   Finder get last => new _LastFinder(this);
 
+  /// Returns a variant of this finder that only matches elements reachable by
+  /// a hit test.
+  ///
+  /// The [at] parameter specifies the location relative to the size of the
+  /// target element where the hit test is performed.
+  Finder hitTestable({@required FractionalOffset at}) => new _HitTestableFinder(this, at);
```

Comment:
maybe make `at` default to `FractionalOffset.center`?

nit: spaces around named arguments

Reply:
Done.